Lib/sysconfig.py

```diff
@@ -112,6 +112,11 @@ def _safe_realpath(path):
 # PC/VS7.1
 if os.name == "nt" and "\\pc\\v" in _PROJECT_BASE[-10:].lower():
     _PROJECT_BASE = _safe_realpath(os.path.join(_PROJECT_BASE, pardir, pardir))
+# PC/VS9.0/amd64
+if (os.name == "nt" 
+   and os.path.basename(os.path.dirname(os.path.dirname(_PROJECT_BASE))).lower() == "pc" 
+   and os.path.basename(os.path.dirname(_PROJECT_BASE)).lower().startswith("vs")):
```

Comment:
"I have very little interest in doing much work to support the old build files (as they are explicitly unsupported)."

Wait wait wait... I fail to follow progress on the Windows build system, so I rely on the devguide which I expect to be up to date, and the guide says:

https://docs.python.org/devguide/setup.html#windows-compiling

"Python 2.7 uses Microsoft Visual Studio 2008, which is most easily obtained through an MSDN subscription."

I have a MSDN account, but after 30 min, I failed to get my MSDN number and I don't know my subscription is still valid or not (well, I guess that, it's expired). It took me 1 hour to find an old ISO of Visual Studio 2008, and not from microsoft.com :-(

I didn't want to install VS 2008, but I just followed the devguide.

... Do you mean that VS 2010 must now be used and the devguide is just outdated?